core/domain/change_domain.py

```diff
@@ -108,13 +116,13 @@ class BaseChange:
     # dict with key as attribute name and value as deprecated values
     # for the attribute.
     # This list can be overriden by subclasses, if needed.
-    ALLOWED_COMMANDS = []
+    ALLOWED_COMMANDS: List[Dict[str, Any]] = []
```

Comment:
Why `Any`?

Reply:
In this snippet
https://github.com/div-yam/oppia/blob/c32b8a07575e624ad6bef751d4e593bc0b4cbce6/core/domain/change_domain.py#L150-L159
`COMMON_ALLOWED_COMMANDS` should have same type annotations as `ALLOWED_COMMANDS`, which is `List[Dict[str, Sequence[str]]`, but using this in that place does not satisfy the operand in 157-158. So, the strictest type annotations I could come up with was `List[Dict[str, Any]]`

Reply:
What is the exact problem on lines 157 and 158?7

Reply:
Mypy threw error that '+' does not satisfy `cmd['required_attribute_names'] + cmd['optional_attribute_names'])` type annotations.

Reply:
Can you please send the exact error?

Reply:
This is not addressed.